tests/baselines/reference/user/bluebird.log

```diff
@@ -162,7 +162,6 @@ node_modules/bluebird/js/release/some.js(133,23): error TS2339: Property 'promis
 node_modules/bluebird/js/release/using.js(78,20): error TS2339: Property 'doDispose' does not exist on type 'Disposer'.
 node_modules/bluebird/js/release/using.js(97,23): error TS2339: Property 'data' does not exist on type 'FunctionDisposer'.
 node_modules/bluebird/js/release/using.js(223,15): error TS2350: Only a void function can be called with the 'new' keyword.
-node_modules/bluebird/js/release/util.js(251,28): error TS2554: Expected 0 arguments, but got 2.
```

Comment:
Improved union calling, probably?

Reply:
Looks very similar to the changes from DT, so :+1: